### PR TITLE
Adds set text action for voice access

### PR DESCRIPTION
**Pull request by chunhtai**
voice access uses set text action to type user spoken sentences. This PR adds support for it.

framework pr: https://github.com/flutter/flutter/pull/77024

https://github.com/flutter/flutter/issues/77271

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I signed the [CLA].
- [ ] All existing and new tests are passing.
- [ ] The reviewer has submitted any presubmit flakes in this PR using the [engine presubmit flakes form] before re-triggering the failure.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[engine presubmit flakes form]: https://forms.gle/Wc1VyFRYJjQTH6w5A

### CODE REVIEW

File: shell/platform/android/io/flutter/view/AccessibilityBridge.java [hunk 1]
```diff
@@ -1056,6 +1056,17 @@ public boolean performAction(
           accessibilityChannel.dispatchSemanticsAction(virtualViewId, Action.DISMISS);
           return true;
         }
+      case AccessibilityNodeInfo.ACTION_SET_TEXT:
+        {
+          // Set text APIs aren't available until API 21. We can't handle the case here so
+          // return false
+          // instead. It's extremely unlikely that this case would ever be triggered in the first
+          // place in API < 21.
```
Comment (justinmc) [inline shell/platform/android/io/flutter/view/AccessibilityBridge.java:1064]: Nit: Some strange line break locations here I think.

**Review by justinmc — APPROVED** [review]
LGTM.  It's cool that we'll be able to support this.

**Comment by chinmaygarde** [comment]
The presubmit failure looks like a flake. I have filed a flake report and re-run the test.

**Comment by chunhtai** [comment]
@chinmaygarde the failure is expected, it needs https://github.com/flutter/flutter/pull/77295

**Pull request merged**